WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -136,6 +139,8 @@ - (void)viewDidLoad
 {
     DDLogMethod();
     [super viewDidLoad];
+    [self.tableView registerNib:MediaQuotaCell.nib forCellReuseIdentifier:@"MediaQuotaCell"];
```

Comment:
Could we have a constant for this identifier, as we use it in multiple places?

Reply:
Sure thing.